main.py

```diff
@@ -0,0 +1,108 @@
+from typedb.client import *
+
+def highestVaccines():
+
+    with TypeDB.core_client("localhost:1729") as client:
+        with client.session("vaccination", SessionType.DATA) as session:
+            with session.transaction(TransactionType.READ) as read_transaction:

```

Comment:
These three lines are repeated multiple times throughout your codebase - can you think of a way to reduce this repetition?

Reply:
Removed them, and opened the session transaction for the complete program.
Is it feasible?